images/singleuser/Dockerfile

```diff
@@ -161,7 +161,7 @@ RUN pip --no-cache-dir install -U pip setuptools wheel
 # Install base notebook packages
 RUN pip install --no-cache-dir \
     jupyterhub==2.1.1 \
-    jupyterlab>=3.2.4 \
+    jupyterlab==3.2.4 \
```

Comment:
nit: Maybe add here the task on why it's pinned or in a comment before RUN or similar.
3.3 was not working either?

Reply:
I hadn't actually tried any of the intermediate versions. And indeed it does seem likely that some of them will work. Since it isn't terribly likely that we'll update to 3.4 or something newer right away. I'll check on how far we can get and put it there for this pr. Good call, thanks!

Reply:
3.3.4 seems to work!